.gitignore

```diff
@@ -8,6 +8,9 @@ conanbuildinfo.txt
 conaninfo.txt
 graph_info.json
 build/
+conan_home/
```

Comment:
Don't think this is a convention - mine is different altogether. If this isn't recommended or mentioned in the Conan client docs, I would remove this from .gitignore.

Remember you can always configure a global gitignore in your profile to achieve the same effect